swagger.yaml

```diff
@@ -0,0 +1,211 @@
+swagger: "2.0"
+info:
+  description: "An API used to query codes for information"
+  version: "1.0.0"
+  title: "ONS Code List API"
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+tags:
+- name: "Code List"
+schemes:
+- "http"
+parameters:
+  id:
+    name: id
+    description: "Id for a list code"
+    in: path
+    type: string
+    required: true
+  codeId:
+    name: codeId
+    description: "CodeId for a discrete value within a code list"
+    in: path
+    type: string
+    required: true
+  pageSize:
```

Comment:
Both new variables should have a default value and become optional fields, (previously we have sat default values for `maxResults` and `offset` to 20 and 0 respectively.